src/Workspaces/CoreTest/SolutionTests/SolutionTests.cs

```diff
@@ -2939,6 +2939,39 @@ public async Task TestChangingAnEditorConfigFile([CombinatorialValues(LanguageNa
             Assert.True(finalCompilation.ContainsSyntaxTree(syntaxTreeAfterEditorConfigChange));
         }
 
+        [Theory]
+        [CombinatorialData]
+        public void TestAddingAndRemovingGlobalEditorConfigFileWithDiagnosticSeverity([CombinatorialValues(LanguageNames.CSharp, LanguageNames.VisualBasic)] string languageName, bool useRecoverableTrees)
+        {
+            using var workspace = useRecoverableTrees ? CreateWorkspaceWithRecoverableSyntaxTrees() : CreateWorkspace();
```

Comment:
You can remove the 'useRecoverableTrees' and make this just a regular [Fact]. The distinction mattered when diagnostic values were held on the trees since we could have bugs where we didn't handle the trees correctly in both cases. It's now moot. I'm guessing this test predates merging the other PR so you were following the correct pattern, at the time.